readme.md

```diff
@@ -124,5 +133,11 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
-
+Fork - create a copy branch of a repo in the users repo.
+Pull Requests - Alerting the master repo of changes done on a fork to be merged into it.
+Clone - Copy a repo locally on a machine.
+Push - Send changes done locally up to a repo branch.
+Branch - Create a separate path of a repo.
+Pull - Bring changes/updates to a repo down to a local machine.
+Merge - Add and combine changes from a branch to another branch or master.
```

Comment:
nice definitions! was looking for "fork and pull" or "single repository" specifically